luigi_pipeline/tests/test_hail_tasks.py

```diff
@@ -90,3 +91,53 @@ def requires(self):
 
         mt = es_task.import_mt()
         self.assertEqual(mt.count(), (30, 16))
+
+    def create_temp_sample_remap_file(self, mt, number_of_remaps):
+        temp_input_file = os.path.join(self.test_dir, 'temp_remap_samples.txt')
+        remap = mt.cols().head(number_of_remaps).key_by('s')
+        remap = remap.annotate(seqr_id=remap.s + '_1')
+        remap.export(temp_input_file, header=True)
+        return temp_input_file
+
+    def create_temp_sample_subset_file(self, mt, number_of_subset_samples, fail=False):
+        temp_input_file = os.path.join(self.test_dir, 'temp_subset_samples.txt')
+        subset = mt.cols().head(number_of_subset_samples)
+        if fail:
+            subset = subset.annotate(test='wrong_sample').key_by('test')
+            subset = subset.drop('s').rename({'test': 's'})
+        subset.export(temp_input_file, header=True)
+        return temp_input_file
+
+    def test_hail_matrix_table_remap_0(self):
+        task = HailMatrixTableTask(source_paths=[TEST_DATA_MT_1KG], dest_path="empty", genome_version='37')
+        mt = hl.import_vcf(TEST_DATA_MT_1KG)
+        remap_mt = task.remap_sample_ids(mt, self.create_temp_sample_remap_file(mt, 0))
+        self.assertEquals(remap_mt.anti_join_cols(mt.cols()).count_cols(), 0)
```

Comment:
`assertEqual` would be preferable here since `assertEquals` is a deprecated alias.

https://docs.python.org/3/library/unittest.html#deprecated-aliases